src/globus_sdk/services/gcs/data/collection.py

```diff
@@ -257,9 +281,10 @@ def __init__(
         public: t.Optional[bool] = None,
         # > common args end <
         # > specific args start <
-        storage_gateway_id: t.Optional[UUIDLike] = None,
         # strs
         domain_name: t.Optional[str] = None,
+        guest_auth_policy_id: t.Optional[UUIDLike] = None,
+        storage_gateway_id: t.Optional[UUIDLike] = None,
```

Comment:
It looks like these type-based comment markers (`# strs`, `# str lists`) are becoming inaccurate.

Reply:
I'm not sure if they're useful or not, but I meant "is a string in the payload" (rather than the types accepted by the SDK helper). So the `storage_gateway_id` moved in part to try to make it accurate again.

Like I said, I'm not really sure if this is useful. I'm just trying to give us some handle for looking through these (long!) lists of arguments.